src/quantum/azext_quantum/tests/latest/utils.py

```diff
@@ -9,12 +9,11 @@
 TEST_WORKSPACE_DEFAULT_LOCATION = "westus2"
 TEST_WORKSPACE_DEFAULT_STORAGE = "e2etests"
 TEST_WORKSPACE_DEFAULT_STORAGE_GRS = "e2etestsgrs"
-TEST_WORKSPACE_DEFAULT_PROVIDER_SKU_LIST = "microsoft-qc/learn-and-develop"
-TEST_CAPABILITIES_DEFAULT = "new.microsoft-qc;submit.microsoft-qc"
-
-TEST_TARGET_DEFAULT_PROVIDER_SKU_LIST = "microsoft-qc/learn-and-develop"
-TEST_TARGET_DEFAULT_PROVIDER = "microsoft-qc"
-TEST_TARGET_DEFAULT_TARGET = "microsoft.estimator"
+TEST_WORKSPACE_DEFAULT_PROVIDER_SKU_LIST = "Microsoft.ChemistryHpc/Basic"
+TEST_CAPABILITIES_DEFAULT = "new.Microsoft.ChemistryHpc;submit.Microsoft.ChemistryHpc"
+TEST_TARGET_DEFAULT_PROVIDER_SKU_LIST = "Microsoft.ChemistryHpc/Basic"
+TEST_TARGET_DEFAULT_PROVIDER = "Microsoft.ChemistryHpc"
+TEST_TARGET_DEFAULT_TARGET = "microsoft.hpc"
```

Comment:
Did this configuration work for making recordings? I think, in order to run tests against DFT (as we run them in `tests_quantum_targets.py`) we'll need to use:
```suggestion
TEST_WORKSPACE_DEFAULT_PROVIDER_SKU_LIST = "microsoft-elements/elements-internal-testing"
TEST_CAPABILITIES_DEFAULT = "new.microsoft-elements;submit.microsoft-elements"
TEST_TARGET_DEFAULT_PROVIDER_SKU_LIST = "microsoft-elements/elements-internal-testing"
TEST_TARGET_DEFAULT_PROVIDER = "microsoft-elements"
TEST_TARGET_DEFAULT_TARGET = "microsoft.dft"
```
Please verify the values above locally.

Reply:
Hi @kikomiss ,
I originally specified “microsoft-elements” as you are suggesting, but when I ran the CLI tests from my branch in the azure-quantum-clients.e2e.validation pipeline, this test failed:
`[gw1] [ 42%] FAILED src\quantum\azext_quantum\tests\latest\test_quantum_targets.py::QuantumTargetsScenarioTest::test_get_provider `

It failed because the e2e tests are running in West US 2, and microsoft-elements is not available in that region:
`E azure.cli.core.azclierror.InvalidArgumentValueError: Provider 'microsoft-elements' not found in region westus2.`
See [Pipelines - Run 20241231.1](https://dev.azure.com/ms-azurequantum/AzureQuantum/_build/results?buildId=84820&view=results)

That’s why I specified “Microsoft.ChemistryHpc” instead of “microsoft-elements”.  The CLI tests passed in [Pipelines - Run 20241231.2](https://dev.azure.com/ms-azurequantum/AzureQuantum/_build/results?buildId=84830&view=results)

Perhaps we should investigate changing the region used in the azure-quantum-clients.e2e.validation pipeline, but I feel that’s out-of-scope for [User Story 29126](https://dev.azure.com/ms-azurequantum/AzureQuantum/_workitems/edit/29126)/[PR 8388](https://github.com/Azure/azure-cli-extensions/pull/8388)

Reply:
I noticed `test_submit_dft` is being skipped in the e2e pipeline:
`[gw0] [ 47%] SKIPPED src\quantum\azext_quantum\tests\latest\test_quantum_jobs.py::QuantumJobsScenarioTest::test_submit_dft`
(See https://dev.azure.com/ms-azurequantum/AzureQuantum/_build/results?buildId=84830&view=logs&j=47d29c41-921d-590a-26f1-c3ad03029a68&t=f2ab45fa-c299-54cb-4070-4ede54dcd486&l=401)

Does this warrant a bug report?